releasenotes/notes/0.19/user-config-mpl-style-a9ae4eb7ce072fcd.yaml

```
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixed a small bug where the circuit drawer would ignore the Matplotlib style
+    specified in ``settings.conf``.
```

```suggestion
    Fixed an issue with the :meth:`~qiskit.circuit.QuantumCircuit.draw` method and
    :func:`~qiskit.visualization.circuit_drawer` where a custom style set via the
    user config file (i.e. ``settings.conf``) would ignore the set value of the
    ```circuit_mpl_style`` field if the ``style`` kwarg on the function/method was not
    set.
```